### PR TITLE
Fix clang errors and warnings

### PR DESCRIPTION
**Pull request by carlodefalco**
Fix clang errors and warnings

### CODE REVIEW

File: src/coder_runtime.cpp [hunk 1]
```diff
@@ -3801,7 +3801,7 @@ namespace coder
                     const octave_value& tmp = rhs_val (k);
 
                     if (tmp.is_undefined ())
-                      error ("element number %d undefined in return list",
+                      error ("element number %lld undefined in return list",
```
Comment (mmuetzel) [inline src/coder_runtime.cpp]: mingw won't understand this format specifier.
Please consider using the PRId64 macro: http://www.cplusplus.com/reference/cinttypes/

Comment (carlodefalco) [reply src/coder_runtime.cpp]: Thanks for pointing this out.
Would it be OK to use :

error ("element number %s undefined in return list", std::to_string (k+1).c_str ());

instead?

Comment (mmuetzel) [reply src/coder_runtime.cpp]: That sounds reasonable to me as well.

**Review by shsajjadi — APPROVED** [review]
Very good, Thanks.

**Review by shsajjadi — COMMENTED** [review]
I think -g0 can be used instead of -gtoggle

**Review by shsajjadi — CHANGES_REQUESTED** [review]
I think -g0 can be used instead of -gtoggle

**Pull request merged**